app/routes/questions/$question.tsx

```diff
@@ -16,13 +16,31 @@ export const loader = async ({request, params}: Parameters<LoaderFunction>[0]) =
     throw Error('missing question title')
   }
 
-  return await loadQuestionDetail(request, question)
+  try {
+    return await loadQuestionDetail(request, question)
+  } catch (error) {
+    return {
+      error: error.toString(),
```

Comment:
this is a slight potential security issue, if there are sensitive stuff in the error message. Should be fine?

Reply:
We only forward reponses from Coda, not the request with keys, and if someone wanted to send a bunch of spam likes, they can misuse the current system without generating any error messages, so I don't believe forwarding errors is a significant risk at this point... We might need a security audit if Stampy becomes more popular though, the feature to allow anonymous access might not survive if we start dealing with trolls or similar.